core/src/main/kotlin/com/github/inflab/spring/data/mongodb/core/aggregation/search/InSearchOperatorDsl.kt

```diff
@@ -0,0 +1,125 @@
+package com.github.inflab.spring.data.mongodb.core.aggregation.search
+
+import com.github.inflab.spring.data.mongodb.core.annotation.AggregationMarker
+import org.bson.Document
+import org.bson.types.ObjectId
+import java.time.temporal.Temporal
+
+/**
+ * A Kotlin DSL to configure the in operator using idiomatic Kotlin code.
+ *
+ * @author cranemont
+ * @since 1.0
+ * @see <a href="https://www.mongodb.com/docs/atlas/atlas-search/in/#in">in</a>
+ */
+@AggregationMarker
+class InSearchOperatorDsl {
+    private val document = Document()
+
+    /**
+     * The indexed field to search.
+     * You can also specify a wildcard path to search.
+     * See path construction for more information.
+     *
+     * @param path Indexed field to search.
+     * @see <a href="https://www.mongodb.com/docs/atlas/atlas-search/path-construction/#std-label-ref-path">Path Construction</a>
+     */
+    fun path(path: String) {
+        document["path"] = path
+    }
+
+    /**
+     * The score assigned to matching search term results. Use one of the following options to modify the score:
```

Comment:
Please insert a new line after the period.

```suggestion
     * The score assigned to matching search term results.
     * Use one of the following options to modify the score:
```